code/datums/supply_packs/clothing.dm

```diff
@@ -0,0 +1,528 @@
+//////CLOTHING//////
+
+/datum/supply_packs/costume
+	name = "Standard costumes"
+	contains = list(/obj/item/weapon/storage/backpack/clown,
+					/obj/item/clothing/shoes/clown_shoes,
+					/obj/item/clothing/mask/gas/clown_hat,
+					/obj/item/clothing/under/rank/clown,
+					/obj/item/weapon/bikehorn,
+					/obj/item/clothing/under/mime,
+					/obj/item/clothing/shoes/mime,
+					/obj/item/clothing/gloves/white,
+					/obj/item/clothing/mask/gas/mime,
+					/obj/item/clothing/head/beret,
+					/obj/item/clothing/suit/suspenders,
+					/obj/item/weapon/reagent_containers/food/drinks/bottle/bottleofnothing,
+					/obj/item/weapon/hair_dye)
+	cost = 10
+	containertype = /obj/structure/closet/crate/secure/basic
+	containername = "standard costumes crate"
+	access = list(access_theatre)
+	group = "Clothing"
+	containsdesc = "A set of replacement clown and mime costumes."
```

Comment:
"A set of replacement clown and mime costumes, for when the originals end up spaced"

Reply:
Igotchu.